core/notifications/src/notification_event/price_changed.rs

```diff
@@ -40,7 +40,7 @@ pub struct PriceChanged {
 
 impl PriceChanged {
     const NOTIFICATION_THRESHOLD: ChangePercentage = ChangePercentage(5.0);
-    const COOL_OFF_PERIOD: chrono::Duration = chrono::Duration::days(2);
+    const COOL_OFF_PERIOD: Option<chrono::Duration> = chrono::Duration::try_days(2);
```

Comment:
I don't understand why this const should be `Option`? We know its never invalid (its a constant) and this type makes it every reference worse... - what's the point of the const in that case?